### PR TITLE
Fix #94

### LINKED ISSUE #94: Error in proctor guide
This page says "Azure Container Registry itself is a free service." however it has a daily cost, of at least $0.17 per day...

https://github.com/microsoft/MCW-Containers-and-DevOps/blob/master/Whiteboard%20design%20session/WDS%20trainer%20guide%20-%20Containers%20and%20DevOps.md

https://azure.microsoft.com/en-us/pricing/details/container-registry/

### PR DESCRIPTION
**Pull request by jamesrcounts**

### CODE REVIEW

**Comment by lastcoolnameleft** [comment]
Hey everyone,

I made an oops and accidentally committed the FabMedical files into the master repo. 
 https://github.com/microsoft/MCW-Containers-and-DevOps/commit/e669a41956d0b77ec658591eea6da8b29e65c986

I've rebased to remove that commit; however, because the generated SHA1 has been changed, the parent commit has changed.

That said, I've gone ahead and verified that the master branch has not lost any commits.  I apologize for any inconvenience.

Old SHA1:
https://github.com/microsoft/MCW-Containers-and-DevOps/commit/ed833acb591df3dbadcd84f7d7cb97a43706a7a2

New SHA1:
https://github.com/microsoft/MCW-Containers-and-DevOps/commit/b2c076a30e995a3ff54add1fb8bb385d58bbb558

TL;DR:  This PR might not show the same parent SHA1; however, I have verified that the commits are definitely there.

**Comment by waltermyersiii** [comment]
Thanks, Tommy. Our intention has always been that our contributors only have Read rights and that only Admins would be able to perform commits. Our instructions have always been to issue any updates through a PR, and to never make direct commits on any of the repos.

**Comment by lastcoolnameleft** [comment]
Thanks for the understanding @waltermyersiii .  I think it would be valuable to enable "Protected Branch Rules" for this repo, which could prevent mistakes like this:
https://help.github.com/en/articles/about-protected-branches

You can enable it by going to "Settings -> Branches - Add Rule"
